test/processor/physical_plan/operator/orderBy/order_by_key_encoder_test.cpp

```diff
@@ -52,9 +52,8 @@ class OrderByKeyEncoderTest : public Test {
         for (auto i = 0u; i < numOfOrderByCols; i++) {
             shared_ptr<ValueVector> valueVector =
                 make_shared<ValueVector>(INT64, memoryManager.get());
-            auto intValuePtr = (int64_t*)valueVector->values;
             for (auto j = 0u; j < numOfElementsPerCol; j++) {
-                intValuePtr[j] = 5;
+                valueVector->setValue(j, (int64_t)5);
```

Comment:
No need to change. Just curious
is setValue<int64_t>(j, 5) not working in this case?

Reply:
It won't. So this is the part that the change can introduce unsafe code with potential bugs.
Let's discuss this in person to go over the options here.